formapp/app.py

```diff
@@ -1854,7 +1924,7 @@ def endpoint_ttt():
 
 
         #If there is no game with these players
-        return redirect("/chat", message="No game found")
+        return redirect("/chat?alert=No game found")
```

Comment:
I prefer having the current message system. The chat system needs and overhaul and we should wait until that's done before relying on it for important messages.

Reply:
This doesn't send a chat message; it takes the user to the chat page (since that's the default in advanced mode) and shows the message as an alert().